generators/upgrade/index.js

```diff
@@ -441,8 +448,11 @@ module.exports = class extends BaseGenerator {
             },
 
             updateJhipster() {
+                if (this.originalTargetJhipsterVersion === 'global') {
```

Comment:
'global' should be store into a class variable and documented as we can't know where this key is come from.